.github/workflows/license-checker.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - name: Check License Header
-        uses: apache/skywalking-eyes/header@4f4281f13ebf854ebf514c1d5c3ea6797ae7e234
+        uses: apache/skywalking-eyes/header@main
```

Comment:
This may be unreliable if the main branch makes a break change. It’s better to lock a fixed commit.

Reply:
done, we need to reduce sth like https://github.com/apache/apisix-dashboard/pull/3037 🤔